src/gui/transferlistwidget.cpp

```diff
@@ -353,7 +352,7 @@ void TransferListWidget::setSelectedTorrentsLocation()
         }
     });
 
-    fileDialog->show();
+    fileDialog->open();
```

Comment:
I think this would be the correct usage pattern regardless if it fixes anything.
There is another instance of this at `MainWindow::askRecursiveTorrentDownloadConfirmation()`: `confirmBox->show();` would you mind change it too?

Reply:
> I think this would be the correct usage pattern regardless if it fixes anything.

👍 


> There is another instance of this at `MainWindow::askRecursiveTorrentDownloadConfirmation()`: `confirmBox->show();` would you mind change it too?

I will do it as soon as the next window of free time appears.

Reply:
> I will do it as soon as the next window of free time appears.

Done.